### PR TITLE
Balloon nonpublic scoreboard

### PR DESCRIPTION
**Pull request by vmcj**
Based on the suggestions of @nickygerritsen in https://github.com/DOMjudge/domjudge/issues/1289.

This does not close that issue as discussion there created more functionality questions.

This discloses a bit more information to balloon runners as in the past they knew nothing in default setting during a frozen scoreboard about the amount of submissions which are now disclosed as pending.

### CODE REVIEW

**Review by nickygerritsen — APPROVED** [review]

**Comment by nickygerritsen** [comment]
I think this is a good solution for now to see at least something useful

**Comment by vmcj** [comment]
I'll try to make some extra tests before merging.

**Comment by vmcj** [comment]
@nickygerritsen can you look at the tests if you agree?

File: webapp/src/DataFixtures/Test/DemoPreDeactivateContestFixture.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+class DemoPreDeactivateContestFixture extends AbstractTestDataFixture
+{
+    public function load(ObjectManager $manager): void
+    {
+        $demoContest = $manager->getRepository(Contest::class)->findOneBy(['shortname' => 'demo']);
+        $demoContest->setEndtimeString(
+            sprintf(
+                '%s-06-06 09:00:00 Europe/Amsterdam',
+                date('Y') -1
```
Comment (nickygerritsen) [inline webapp/src/DataFixtures/Test/DemoPreDeactivateContestFixture.php:16]: Nit: missing space

**Review by nickygerritsen — APPROVED** [review]

File: webapp/src/DataFixtures/Test/DemoPreDeactivateContestFixture.php (hunk 1 above)
Comment (vmcj) [reply webapp/src/DataFixtures/Test/DemoPreDeactivateContestFixture.php]: Fixed.

**Pull request merged**